ppl/cmd/zqd/listen/command.go

```diff
@@ -147,7 +150,11 @@ func (c *Command) Run(args []string) error {
 			return err
 		}
 	}
-	return srv.Wait()
+	if core.IsTemporalWorker() {
+		g.Go(func() error { return core.RunTemporalWorker(ctx) })
+	}
+	g.Go(srv.Wait)
+	return g.Wait()
```

Comment:
IMHO I think I'd rather have core.New launch the temporal go routine and have the call to core.Shutdown exit the go-routine.

Reply:
Done in 17c20124e8a4d5f97b496b34b532153f0ba75ffa.